rust-runtime/aws-smithy-http-server-python/src/error.rs

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//! Python error definition.
+
+use thiserror::Error;
+
+/// Python error that implements foreign errors.
+#[derive(Error, Debug)]
+pub enum Error {
+    /// Custom error.
+    #[error("{0}")]
+    Custom(String),
+    /// Implement `From<pyo3::PyErr>`.
```

Comment:
Are the Rustdoc comments accurate? Starting with "Implement" I mean.

Reply:
Regarding `thiserror`, why are we not using it in other places? @rcoh

Reply:
Fixed Rustdoc comments.